src/request.js

```diff
@@ -28,12 +28,17 @@ function isRequest(input) {
 	);
 }
 
-// TODO: use toString check after https://github.com/mysticatea/abort-controller/pull/5
-function isAbortSignal(input) {
-	return (
-		typeof input === 'object' &&
-		input instanceof AbortSignal
-	);
+let isAbortSignal;
+if (typeof Symbol === "function" && typeof Symbol.toStringTag === "symbol") {
+	isAbortSignal = function(input) { return input.toString() === "[object AbortSignal]" };
```

Comment:
This fails for Node v4.

Reply:
This makes me think we should stop supporting Node.js v4.x if we want this feature.